test/scripts/cloudformation-template.coffee

```diff
@@ -8,7 +8,7 @@ appRoot = "test/data/test-app/"
 env = "test"
 
 call ->
-  config = yield configuration.read appRoot
+  config = yield configuration.readApp appRoot
```

Comment:
Kind of wondering if the need for `appRoot` here is suspect. We also have it in the code all over the place. Maybe that should be a default?